core/mutations/CreateArticle.js

```diff
@@ -0,0 +1,10 @@
+import gql from "graphql-tag";
+
+export default gql`
+  mutation createArticle($title: String!, $parentId: ID) {
```

Comment:
Naming convention for  operation name in SDL is CamelCase so this should be `CreateArticle`